src/test/java/com/epam/tc/hw2/GeneralData.java

```diff
@@ -0,0 +1,9 @@
+package com.epam.tc.hw2;
+
+public class GeneralData {
+    protected  String indexUrl = "https://jdi-testing.github.io/jdi-light/index.html";
+    protected  String expectedBrowserTitle = "Home Page";
```

Comment:
Why did you add extra space?

Reply:
Hmm, to be honest, I didn't think about allocating space. Tell me, should I make them private? And getter methods. Or make it public?